app/util/searchUtils.js

```diff
@@ -16,32 +16,6 @@ import mapPeliasModality from './pelias-to-modality-mapper';
 import { PREFIX_ROUTES, PREFIX_STOPS } from './path';
 import { getConfiguration } from '../config';
 
-function useKyytiGeocoding(config) {
-  return (
-    !config.URL.GEOCODING_BASE_URL.includes('digitransit') &&
-    !config.URL.GEOCODING_BASE_URL.includes('mock_api_url')
-  );
-}
-
-function mapKyytiAddrToFeature(res) {
-  return {
-    type: 'Feature',
-    geometry: {
-      type: 'Point',
-      coordinates: [res.location.lon, res.location.lat],
-    },
-    properties: {
-      gid: res.gid,
-      layer: 'locality',
-      name: res.name,
-      country: res.country,
-      country_a: res.country,
-      locality: res.city,
-      label: res.title,
-    },
-  };
-}
```

Comment:
All the Kyyti geocoding stuff in this file shouldn't be deleted? At least for me now the geocoding on the main route planner webpage does not work. The widget stuff shouldn't be working, but does and looks to be connecting to digitransit api instead of our own geocoding API. I looked at this in the network tab of the browser developer console to confirm. That needs to be fixed, so that we use our own geocoding api in the widget too.